pkgs/tools/security/bitwarden/cli.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, buildNpmPackage
+, nodejs_18
+, fetchFromGitHub
+, python3
+}:
```

```suggestion
{ lib
, stdenv
, buildNpmPackage
, nodejs_18
, fetchFromGitHub
, python3
, darwin
}:
```

Add required arguments `stdenv` and `darwin` for above review item